ilastik/widgets/stackFileSelectionWidget.ui

```diff
@@ -110,6 +110,13 @@ Examples:
        </layout>
       </widget>
      </item>
+     <item>
+      <widget class="QLabel" name="import_copy_info">
+       <property name="text">
+        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Note: ilastik will copy the data into the project (.ilp) file.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+       </property>
+      </widget>
```

Comment:
Italicising the font can be done without parsing a whole HTML doc; is there a benefit to doing it that way?
```suggestion
      <widget class="QLabel" name="import_copy_info">
       <property name="text">
        <string>Note: ilastik will copy the data into the project (.ilp) file.</string>
       </property>
       <property name="font">
        <font><italic>true</italic></font>
       </property>
      </widget>
```

Reply:
that's what QtDesigner produced 🤷 , but sure if there is a simpler way to do it...